v5/releases/kernel3.2.0.rst

```diff
@@ -0,0 +1,39 @@
+=========================
+PROS Kernel 3.2.0 Release
+=========================
+
+.. post:: 7 September, 2019
+   :tags: blog, kernel-release
+
+New Features:
+
+- Upgrade LVGL to version 5.3
+- Add an alternate :code:`pros::Task` constructor that takes only a task function, parameters, and a name.
```

Comment:
Not an actual issue, but for future reference you can use double backticks around inline code instead of the more verbose `:code:` preface

Reply:
🤷‍♀ I change what I use depending on my mood when writing these. might be worth auditing this repo for style in the future (and actually making up a styleguide)